use_cases.md

```diff
@@ -227,8 +225,8 @@ no
 Warning: resource 'nodes' is not namespace scoped
 no
 
-# kubectl auth can-i get persistentvolumes
-Warning: resource 'persistentvolumes' is not namespace scoped
+# kubectl auth can-i get persistent volumes
```

Comment:
`persistentvolumes` is not a typo, but rather a Kubernetes specific resource name, please revert.

Reply:
@YashThakare please, fix this.